packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -168,6 +168,10 @@ export function getSlottedContent(
     node: IrLwcComponent | IrComponent,
     cxt: TransformerContext
 ): EsStatement[] {
+    const isNested = cxt.isSlotted;
```

Comment:
```suggestion
    const { isSlotted } = cxt;
```

Nitpick, but let's keep the naming consistent. 🙂